flash/image/segmentation/viz.py

```diff
@@ -60,6 +69,10 @@ def _show_images_and_labels(self, data: List[Any], num_samples: int, title: str)
         fig, axs = plt.subplots(rows, cols)
         fig.suptitle(title)
 
+        if not isinstance(axs, np.ndarray):
+            axs = np.array(axs)
+        axs = axs.flatten()
+
         for i, ax in enumerate(axs.ravel()):
```

Comment:
This will be redundant, so worth removing this here and relying on `axs.flatten()` call that you added in your PR. :)

```suggestion
        for i, ax in enumerate(axs):
```